src/app/_components/DesktopNavigation.tsx

```diff
@@ -0,0 +1,181 @@
+      aria-label={`${label} (opens a new window)`}
+    >
+      <div className="flex gap-1">
+        <p className="pb-2 font-bold">{label}</p>
+        <span className="text-brand-400 group-hover:text-brand-100">
+          <Icon component={ArrowUpRight} size={20} />
+        </span>
+      </div>
+
+      <p className="text-brand-300">{description}</p>
+    </a>
+  )
+}
+
+const getInvolvedInternalItems = getInvolvedItems.filter((item) =>
+  item.href.startsWith('/'),
+)
+
+const getInvolvedExternalItems = getInvolvedItems.filter(
+  (item) => item.href.startsWith('https://') || item.href.startsWith('http://'),
+)
+
+export function DesktopNavigation() {
+  const pathname = usePathname()
+
+  return (
+    <ul
+      className="relative z-10 hidden items-center gap-0.5 lg:flex"
+      aria-label="Navigation items"
+    >
+      <li>
+        <Link
+          href={PATHS.ABOUT.path}
+          className={clsx(
+            'text-hover:bg-brand-700 inline-block rounded-xl px-4 py-1.5 focus:outline focus:outline-2 focus:outline-brand-100',
+            pathname.startsWith(PATHS.ABOUT.path)
+              ? 'text-brand-400'
+              : 'text-brand-300',
+          )}
+        >
+          {PATHS.ABOUT.label}
+        </Link>
+      </li>
+
+      <NavigationPopover as="li" label="Get Involved">
+        <div className="grid w-screen max-w-2xl grid-cols-2 gap-4">
+          <div className="space-y-4">
+            {getInvolvedInternalItems.map((item) => (
+              <InternalLink key={item.href} {...item} />
+            ))}
+          </div>
+          <div className="space-y-4">
+            {getInvolvedExternalItems.map((item) => (
+              <ExternalLink key={item.href} {...item} />
+            ))}
+            <a
+              href={learnMoreItem.href}
+              target="_blank"
+              rel="noopener noreferrer"
+              className="group inline-flex w-full justify-center gap-1 rounded-lg bg-brand-800 px-3 py-5 hover:bg-brand-700 focus:outline focus:outline-2 focus:outline-brand-100"
+              aria-label={`${learnMoreItem.label} (opens a new window)`}
+            >
+              <span className="font-bold text-brand-300">
+                {learnMoreItem.label}
+              </span>
+              <span className="text-brand-400 group-hover:text-brand-100">
+                <Icon component={ArrowUpRight} size={20} />
+              </span>
+            </a>
+          </div>
+        </div>
+      </NavigationPopover>
+
+      <NavigationPopover as="li" label="Community">
+        <div className="w-80 space-y-4">
+          {communityItems.map((item) => (
+            <InternalLink key={item.label} {...item} />
+          ))}
+        </div>
+      </NavigationPopover>
+
+      <li>
+        <Link
+          href={PATHS.BLOG.path}
+          className={clsx(
+            'inline-block rounded-xl px-4 py-1.5 text-brand-300 hover:bg-brand-700 focus:outline focus:outline-2 focus:outline-brand-100',
```

Comment:
```suggestion
            'inline-block rounded-xl px-4 py-1.5 hover:bg-brand-700 focus:outline focus:outline-2 focus:outline-brand-100',
```

Reply:
Because we're setting it below:

```
pathname.startsWith(PATHS.BLOG.path)
              ? 'text-brand-400'
              : 'text-brand-300',
```

But this may become irrelevant [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/118/files#r1599913830)